source/uwp/Renderer/lib/AdaptiveChoiceSetInputRenderer.cpp

```diff
@@ -42,4 +73,176 @@ namespace AdaptiveNamespace
             jsonObject, elementParserRegistration, actionParserRegistration, adaptiveWarnings, element);
     }
     CATCH_RETURN;
+
+    static std::vector<std::string> GetChoiceSetValueVector(_In_ IAdaptiveChoiceSetInput* adaptiveChoiceSetInput)
```

Comment:
>GetChoiceSetValueVector [](start = 36, length = 23)

Should these also be on the renderer class? I realize there's no state on these renderer classes, but these methods should probably conceptually be treated the same as the _Build*ChoiceSetInput methods.

Reply:
yeah, probably :D

---
In reply to: [325832661](https://github.com/microsoft/AdaptiveCards/pull/3457#discussion_r325832661) [](ancestors = 325832661)